applications/crossbar/doc/system_status.md

```diff
@@ -0,0 +1,20 @@
+# System Status
+
+## About System Status
+
+Helps to check cluster's health over crossbar
+
+## Schema
+
+
+
+## Fetch
+
+> GET /v2/system_status
+
+```shell
+curl -v -X GET \
+    -H "X-Auth-Token: {AUTH_TOKEN}" \
+    http://{SERVER}:8000/v2/system_status
+```
+
```

Comment:
Two things:

1. Can you add a sample (anonymized) JSON response payload
2. Can you add a warning that this API requires a superduper admin auth token?

Reply:
Hi @jamesaimonetti! Comments have been addressed.